lib/src/solver/report.dart

```diff
@@ -130,6 +130,39 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    var outdatedPackagesCount = 0;
```

Comment:
Instead of counting this way we could just do:

`final outdatedPackagesCount = _result.packages.where((id) {.......}).length;`

It worked rather nicely for `numChanged` above...

Reply:
Updated the logic.